gate-web/src/main/groovy/com/netflix/spinnaker/gate/config/GateConfig.groovy

```diff
@@ -150,23 +145,21 @@ class GateConfig extends RedisHttpSessionConfiguration {
   @Autowired
   ServiceConfiguration serviceConfiguration
 
+  @Autowired
+  Jackson2ObjectMapperBuilder objectMapperBuilder
+
   /**
    * This needs to be before the yaml converter in order for json to be the default
    * response type.
    */
   @Bean
   AbstractJackson2HttpMessageConverter jsonHttpMessageConverter() {
-    ObjectMapper objectMapper = new ObjectMapper()
-      .enable(DeserializationFeature.READ_UNKNOWN_ENUM_VALUES_AS_NULL)
-      .disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES)
-      .registerModule(new JavaTimeModule())
-
-    return new JsonHttpMessageConverter(objectMapper)
+    return new JsonHttpMessageConverter(objectMapperBuilder.build())
```

Comment:
I don't know how many times I have to emphasize this, but Spring Boot offers this super useful builder class for making an `ObjectMapper`, and you can configure default settings in the Spring properties somewhere. Thus, I've moved the feature flags to `application.properties` on the classpath which Spring Boot picks up by default.

Reply:
At least in this commit, I don't see anything moving to application.properties.  In [this other commit](https://github.com/spinnaker/gate/pull/1663/commits/0e466cf28dedddaa584f5554e36ad6f798b42e32#diff-5589437e0d2e6181d677f0d3066b531d865168d2e32ce89cc0c80f71ecde51baR18) I do see `read-unknown-enum-values-as-null` set to true.  I don't see an equivalent of `.disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES)` nor `.registerModule(new JavaTimeModule()` though.

Reply:
Disabling fail on unknown properties is a default setting in `Jackson2ObjectMapperBuilder`. `JavaTimeModule` is one of the "standard" Jackson modules that class looks for, too. See https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/http/converter/json/Jackson2ObjectMapperBuilder.html for more info.